wp-content/themes/csisnuclearnetwork/assets/_scss/components/_post-block-related.scss

```diff
@@ -6,10 +6,25 @@
   margin-left: auto;
   text-align: center;
 
+  &__subtitle {
+    text-transform: uppercase;
+    a {
+      color: $color-accent-primary-200;
+    }
+  }
```

Comment:
Once you do the rebase, this can be removed as it's handled in `_post-meta.scss`.